### DISCUSSION TITLE
Lit overlay, make selected way more clear?

### DISCUSSION THREAD

**Discussion by ivanbranco** [streetcomplete/StreetComplete#4707] [Feedback]
In the lit overlay, if I select part of a way, I cannot understand how long the way I selected is, since there's no difference with the rest of the way.

An example, if I select a "green way" it's obvious what I'm editing:
![image](https://user-images.githubusercontent.com/94002623/209727478-881be71c-bf84-463f-9100-657f7ca8c0f7.png)

if I select a "red way" it's way more difficult to understand where the way starts/ends:
![image](https://user-images.githubusercontent.com/94002623/209727546-27d2ac32-7b23-46e7-bd47-304022ffdb43.png)

I often don't reply to this quest because I don't want to undone the edits (and make the object advance by two versions creating unnecessary noise in the history of it).

**Comment by matkoniecz** [comment]
See #4417
